go/internal/sidecar/sidecar_test.go

```
@@ -0,0 +1,557 @@
+			t.Fatalf("path header = %q", r.Header.Get("X-Ocx-Go-Sidecar-Path"))
+		}
+		if r.URL.RawQuery != "limit=1" {
+			t.Fatalf("query = %q", r.URL.RawQuery)
+		}
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusServiceUnavailable)
+		_, _ = w.Write([]byte(`{"error":{"code":"lab_projection_unavailable"}}`))
+	}))
+	defer bridge.Close()
+	h := NewHandler(Config{ParentURL: bridge.URL, BridgeToken: bridgeToken, RequestToken: requestToken})
+	missing := do(t, h, http.MethodGet, "/api/lab/verdicts?limit=1")
```

golangci-lint (2.13.2)

[error] response body must be closed

(bodyclose)